### PR TITLE
Consumer API: Get single deletion process

### PR DESCRIPTION
**Pull request by MH321Productions**
# Readiness checklist

-   [ ] I added/updated unit tests.
-   [ ] I added/updated integration tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

### CODE REVIEW

File: Modules/Devices/src/Devices.Application/Identities/Queries/GetDeletionProcess/Handler.cs
Comment (NikolaVetnic) [inline Modules/Devices/src/Devices.Application/Identities/Queries/GetDeletionProcess/Handler.cs:24]: This approach certainly works, but maybe it's a bit of an overkill to fetch the entire identity and all its deletion processes just to get the one you need... On the other hand, I can imagine that there will rarely be an identity with more than one process, and probably never with a significant number of them.

I will not request changes here so that @tnotheis may have the final word on this.

File: Modules/Devices/src/Devices.ConsumerApi/Controllers/IdentitiesController.cs
Comment (NikolaVetnic) [inline Modules/Devices/src/Devices.ConsumerApi/Controllers/IdentitiesController.cs:88]: Maybe you would also need integration tests for this but I suppose we should wait for decision of @tnotheis on this one.

**Review by NikolaVetnic — APPROVED** [review]

File: Modules/Devices/src/Devices.Application/Identities/Queries/GetDeletionProcess/Handler.cs
Comment (tnotheis) [reply Modules/Devices/src/Devices.Application/Identities/Queries/GetDeletionProcess/Handler.cs:24]: I agree that it is not ideal. But on the other hand it's as you said: there won't be that much data, so I don't think it is worth the time we spend on implementing something better. 
Also, we would have to pollute the Identity class as well as the IdentityRepository to achieve this.
So in short: we will leave it as it is.

**Review by tnotheis — APPROVED** [review]

**Pull request merged**
